reference/forms/types/options/datetime_choice_translation_domain.rst.inc

```diff
@@ -0,0 +1,15 @@
+choice_translation_domain
```

Comment:
I don't understand, I've just copy-pasted the current description and changed the version added.